storage/azure/package.json

```diff
@@ -30,10 +30,12 @@
     "test:unit": "npm run test:unit:backend && npm run test:unit:frontend",
     "test:unit:frontend": "cypress run --browser chrome --config-file node_modules/@itwin/object-storage-common-config/unit.cypress.config.ts",
     "test:unit:frontend:debug": "cypress open --browser chrome --config-file node_modules/@itwin/object-storage-common-config/unit.cypress.config.ts",
-    "test:unit:backend": "npm run test:unit:backend:common && npm run test:unit:backend:local",
-    "test:unit:backend:local": "mocha lib/test/unit/backend/**/*.test.js --color --exit",
+    "test:unit:backend": "npm run test:unit:backend:common && npm run test:unit:backend:azureSpecific",
+    "test:unit:backend:azureSpecific": "mocha lib/test/unit/backend/**/*.test.js --color --exit",
     "test:unit:backend:common": "node lib/test/unit/backend/RunCommonUnitTests.js",
-    "test:integration:backend": "node lib/test/integration/backend/RunIntegrationTests.js",
+    "test:integration:backend": "npm run test:integration:backend:common && npm run test:integration:backend:local",
+    "test:integration:backend:local": "mocha lib/test/integration/backend/**/*.test.js --color --exit -t 30000",
```

Comment:
you renamed unit test scripts but integration tests script remain the same?

Reply:
Reverted back to `local` as discussed